### PR TITLE
Changes to joystick channels and field centric tuning

### PR DESCRIPTION
**Pull request by AidanSmith**
I realized that we had both a FloatStatus and a FloatOutput for joysticks, so I simplified it by removing the FloatOutput. This should have no functional change. If you see some functional change, tell me. Also, defaulting to driving in field centric is now tunable.

### CODE REVIEW

**Comment by vincent-miller** [comment]
Looks good

File: QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java [hunk 1]
```diff
@@ -7,15 +7,10 @@
...
-    private static FloatStatus rightJoystickChannelY = new FloatStatus();
-
-    public static FloatOutput leftJoystickX = leftJoystickChannelX;
-    public static FloatOutput leftJoystickY = leftJoystickChannelY;
-    public static FloatOutput rightJoystickX = rightJoystickChannelX;
-    public static FloatOutput rightJoystickY = rightJoystickChannelY;
+    public static final FloatStatus leftJoystickX = new FloatStatus();
+    public static final FloatStatus leftJoystickY = new FloatStatus();
+    public static final FloatStatus rightJoystickX = new FloatStatus();
+    public static final FloatStatus rightJoystickY = new FloatStatus();
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java:13]: The reason for not doing it like this is separation of concerns - not letting users of DriveCode read the values, because these are supposed to be `set`, and allowing reading could be confusing.
However, I'm not convinced that this is critical in this case, so you don't need to revert this change.
